hack/local-development/start-gardenlet

```diff
@@ -17,7 +17,9 @@
 source $(dirname "${0}")/common/helpers
 source $(dirname "${0}")/common/local-imagevector-overwrite
 
-REPO_ROOT=$(dirname $0)/../..
+REPO_ROOT=$(dirname "${0}")/../..
```

Comment:
Please adapt https://github.com/gardener/gardener/blob/master/docs/development/local_setup.md#run-the-gardener accordingly.